### ISSUE TITLE
Upgrade from Chromium 121 to Chromium 122

### ISSUE THREAD

**Issue by emerick** [brave/brave-browser#35200]
Upgrade from Chromium 121 to Chromium 122

https://chromium.googlesource.com/chromium/src/+log/121.0.6167.139..122.0.6261.18/?pretty=fuller&n=10000

**QA tests:**
This is a major Chromium version bump, please do full passes.

**Desktop Affected areas:**
* Verify that tab dragging works as expected
* Verify that default search engine list looks correct
* Verify that permission request bubble works as expected
* Verify that omnibox works as expected
* Verify that syncing works as expected
* Verify that content settings work as expected
* Verify installer and autoupdates, particularly on Mac
* Verify cookie settings/behavior
* Verify Speedreader works as expected
* Verify PDF content extraction (AI feature) works as expected

**iOS Affected areas:**
* Verify bookmarks functionality

**Android Affected areas:**
* NTP layout (double check there is no UI regression)
* VPN (double check you can subscribe and use it)
* Wallet (double check there is no regression)
* Playlist (double check there is no regression (if applicable))
* Widevine support (double check it works as expected)
* Brave Wallet (generally check it works as expected)
* Autoplay setting (double check it works as expected)
* Privacy setting (double check there are no excessive items)
* Private tab (make sure no UI regression)

**Comment by mherrmann** [comment]
**To QA:** Please thoroughly test updates on macOS, including delta updates. Upstream removed large chunks of the implementation we base on, and @emerick had to reinstate it.

**Comment by GeetaSarvadnya** [comment]
Verification INPROGRESS on

```
Brave | 1.64.64 Chromium: 122.0.6261.29 (Official Build) nightly (64-bit)
-- | --
Revision | 2950cdfba24ed3fe3afbeed482c75f229b02f9f9
OS | Windows 10 Version 22H2 (Build 19045.3930)

```

<details>
<summary>Verify that tab dragging works as expected</summary>

</details>

<details>
<summary>Verify that default search engine list looks correct</summary>

</details>

<details>
<summary>Verify that permission request bubble works as expected</summary>

</details>

<details>
<summary>Verify that omnibox works as expected</summary>

</details>

<details>
<summary>Verify that syncing works as expected</summary>

</details>

<details>
<summary>Verify that content settings work as expected</summary>

</details>

<details>
<summary>Verify cookie settings/behavior</summary>

</details>

<details>
<summary>Verify Speedreader works as expected</summary>

</details>

<details>
<summary>Verify PDF content extraction (AI feature) works as expected</summary>

</details>

------
**_Settings comparison_**

<details>



</details>

**_Brave startup calls_**

- Confirmed that only brave domains are listed in the Charles when the nightly build is launched with a clean profile
- Confirmed no calls to google, gstatic, etc domains when the nightly build is launched with a clean profile

**Comment by LaurenWags** [comment]
Verification in progress with

```

```

<details>
<summary>C121 --> C122 on-demand upgrade check - InProgress</summary>

1. Install `C121` build via https://github.com/brave/brave-browser/releases/tag/v1.64.61
2. Launch via terminal as follows: `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates.brave.com/sparkle/Brave-Browser/nothanks/appcast.xml` (this is so you don't pick up any newer Nightly versions that have already been uploaded to sparkle)
3. Watch logs, confirm no update found (can also visit `brave://settings/help` but close this tab prior to closing the browser)
4. Close `Brave`
5. Relaunch with `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr`
6. Navigate to `brave://settings/help` and ensured that an update has been found/being applied

```

```

7. Confirm you are prompted to "Relaunch" to get update (see note in first case about this button)

SS

Once relaunched, ensured the correct version was installed as per the following:

**`Example`** | **`Example`**
----------------|----------------

</details>

**Comment by LaurenWags** [comment]
Verification in progress on macOS arm64 14.x with

```

```

Encountered https://github.com/brave/brave-browser/issues/36013 while testing.

#### Clean install of C122 builds:

<details>
<summary>Install C122 arm64 .dmg - InProgress</summary>

1. Install `C122` arm64 .dmg build via https://github.com/brave/brave-browser/releases/tag/v1.64.65
2. Launch browser
3. Confirm no issues/crashes on launch
4. Close and relaunch
5. Confirm no issues on relaunch
6. Confirm griffin is pulled


-- | --

</details>

<details>
<summary>Install C122 arm64 .pkg - TBD</summary>

1. Install `C122` arm64 .pkg build via https://github.com/brave/brave-browser/releases/tag/v1.64.65, following prompts as needed
2. Launch browser
3. Confirm no issues/crashes on launch
4. Close and relaunch
5. Confirm no issues on relaunch
6. Confirm griffin is pulled


-- | --

</details>

<details>
<summary>Install C122 universal .dmg - TBD</summary>

1. Install `C122` universal .dmg build via https://github.com/brave/brave-browser/releases/tag/v1.64.65
2. Launch browser
3. Confirm no issues/crashes on launch
4. Close and relaunch
5. Confirm no issues on relaunch
6. Confirm griffin is pulled


-- | --

</details>

<details>
<summary>Install C122 universal .pkg - TBD</summary>

1. Install `C122` universal .pkg build via https://github.com/brave/brave-browser/releases/tag/v1.64.65, following prompts as needed
2. Launch browser
3. Confirm no issues/crashes on launch
4. Close and relaunch
5. Confirm no issues on relaunch
6. Confirm griffin is pulled

-- | --

</details>

**Comment by hffvld** [comment]
### IN PROGRESS...

Verified on `Pixel 7` using version(s):
```
Device/OS: Pixel 7 / panther_beta-user 14 AP21.240119.009 release-keys
Brave build: 1.64.69 
Chromium: 122.0.6261.29 (Official Build) canary (64-bit) 
```
-------------------------

<details>
<summary> NTP layout (double check there is no UI regression) </summary>

Confirmed that `NTP` looks correct for both `Standard` and `Private` tabs

|1|2|3|4|
|:-:|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/4447ce6d-deba-47f1-a7ca-a9ab0f743731) | ![2](https://github.com/brave/brave-browser/assets/128532965/7ddd1ae1-af77-403b-8317-46db28011a72) | ![3](https://github.com/brave/brave-browser/assets/128532965/36b811bc-a894-4cd1-8bf3-0e207cef797b) | ![4](https://github.com/brave/brave-browser/assets/128532965/c81dd009-10a9-43fb-bb72-6a9bab3e5cae) |
| ![1](https://github.com/brave/brave-browser/assets/128532965/491828b4-b0cc-4a9f-a73f-8a1c18273a12) | ![2](https://github.com/brave/brave-browser/assets/128532965/9ec5dd17-827a-4d56-83e8-3c27084eeef0) | ![3](https://github.com/brave/brave-browser/assets/128532965/09852f9a-dcb9-45a4-8bfc-a44d9b57d3be) | ![4](https://github.com/brave/brave-browser/assets/128532965/1831ad48-dc3f-4062-aa21-2e1ef55ae80b) |
</details>

<details>
<summary> VPN (double check you can subscribe and use it) </summary>

Confirmed that `VPN` is working, able to purchase a new subscription, restore existing one, and cancel a subscription

|1|2|3|4|
|:-:|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/751e3ef8-1a20-43f6-a731-6115dc7f4487) | ![2](https://github.com/brave/brave-browser/assets/128532965/404e0f75-4421-49b5-98fc-19aac53363ba) | ![3](https://github.com/brave/brave-browser/assets/128532965/8615268b-7769-4775-9fc5-d83f9e406e53) | ![4](https://github.com/brave/brave-browser/assets/128532965/0d7bad1c-0fe3-4b61-99e7-d6ec2970a4a0) |
| ![1](https://github.com/brave/brave-browser/assets/128532965/7d3c3990-7394-4a54-a645-e3cc1ca6d6d4) | ![2](https://github.com/brave/brave-browser/assets/128532965/29ba064d-074b-4c54-8bd6-a0bc3e781630) | ![3](https://github.com/brave/brave-browser/assets/128532965/3343a8fe-c6b4-4b9e-b891-b00f3510812a) | ![4](https://github.com/brave/brave-browser/assets/128532965/4484a099-63d5-437e-99f1-471a58f84ce3) |
</details>

<details>
<summary> Wallet (double check there is no regression) </summary>

Confirmed `Wallet` creation and restoration are working

|1|2|3|4|
|:-:|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/fd2ac006-9981-491c-907a-0ecdfde68e54) | ![2](https://github.com/brave/brave-browser/assets/128532965/91e95af1-d748-46e5-b687-968ba4f93fa8) | ![3](https://github.com/brave/brave-browser/assets/128532965/4671f184-9d7f-4d42-ac47-b891cad3082a) | ![4](https://github.com/brave/brave-browser/assets/128532965/5cc58729-91f4-4cbb-b3ea-c3c07b67b4de) |
</details>

<details>
<summary> Playlist (double check there is no regression (if applicable)) </summary>

Confirmed that the main functionality is working: I can add new videos, play, sort and delete videos, create new playlists, and move videos between playlists

|1|2|3|
|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/218ebc91-8209-477a-a046-f1a14dd406ce) | ![2](https://github.com/brave/brave-browser/assets/128532965/ce6e7353-5dae-42ba-bc9a-c6af5d974b89) | ![3](https://github.com/brave/brave-browser/assets/128532965/4929be78-1ff0-436b-a845-b661d4ffc44c) |
| ![1](https://github.com/brave/brave-browser/assets/128532965/b197e7e4-3a2a-447a-9a94-f6d6ca8d5d7b) | ![2](https://github.com/brave/brave-browser/assets/128532965/7fae59fe-cad9-47c2-9c50-ceba3f84e44d) | ![3](https://github.com/brave/brave-browser/assets/128532965/06b34f61-22b3-4df3-a798-d05c339a27cb) |
| ![1](https://github.com/brave/brave-browser/assets/128532965/e8e2621b-1166-40ae-825f-72e87bbad075) | ![2](https://github.com/brave/brave-browser/assets/128532965/77c5a047-6398-45b0-9467-9e416b725bb8) | ![3](https://github.com/brave/brave-browser/assets/128532965/25f6cf37-4b1b-4949-b3b3-9d3eab36241a) |
</details>

<details>
<summary> Widevine support (double check it works as expected) </summary>

Confirmed that media files on websites like Spotify or HBO Max can be played when `Wedivine` is ON

|1|2|
|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/5328a0b4-293b-41e0-8320-cb6e23aca3cb) | ![2](https://github.com/brave/brave-browser/assets/128532965/5a1ccc56-cc0e-4b31-836c-1de87573e353) |
| ![1](https://github.com/brave/brave-browser/assets/128532965/56b6f824-99ed-4800-a45a-ceac5b386c87) | ![2](https://github.com/brave/brave-browser/assets/128532965/3c3def68-bcd0-4fbb-b23a-c0fadff83790) |
| ![1](https://github.com/brave/brave-browser/assets/128532965/0a683d8b-074e-40ca-986e-87d6c9b781b7) | ![2](https://github.com/brave/brave-browser/assets/128532965/fdc374e2-50b8-4491-bb69-54b613f5d5cc) |
</details>

<details>
<summary> Brave Wallet (generally check it works as expected) </summary>

Confirmed that `Brave Rewards` can be enabled and `Uphold` wallet can be connected

|1|2|3|4|
|:-:|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/2e3e7272-8871-4c27-ae02-c7bcdb599a9c) | ![2](https://github.com/brave/brave-browser/assets/128532965/75b95afa-54e5-49c7-be1b-6c8dd9fae3c5) | ![3](https://github.com/brave/brave-browser/assets/128532965/debf43fa-02c7-48a8-a9e0-2471103a4752) | ![4](https://github.com/brave/brave-browser/assets/128532965/6d61d127-58c8-46da-beef-564a4531e16a) |
| ![1](https://github.com/brave/brave-browser/assets/128532965/c6b359bc-d43b-47bd-9738-38c6ab4a0143) | ![2](https://github.com/brave/brave-browser/assets/128532965/bfef6d38-936a-4b54-888c-6368a9e8ca6e) | ![3](https://github.com/brave/brave-browser/assets/128532965/c5bc2d40-1f0f-4b8a-8872-a0d3cb51961b) | ![4](https://github.com/brave/brave-browser/assets/128532965/99cc1b4f-3502-49cb-9055-613e67eedee9) |
</details>

<details>
<summary> Autoplay setting (double check it works as expected) </summary>

- Confirmed that `Autoplay` next file in the Playlist is not working when the toggle switch is OFF and working when it's ON
- Confirmed that `Background` play is also working as expected when the toggle switch is ON or OFF

|1|2|
|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/a0a5bd2e-a57e-4085-82af-b34480a9b0af) | ![2](https://github.com/brave/brave-browser/assets/128532965/3f96ba09-dfab-4da7-b122-c2e31387d00d) |

https://github.com/brave/brave-browser/assets/128532965/25f7b498-7796-4da5-8d4d-b6a6e6fb5438

https://github.com/brave/brave-browser/assets/128532965/c075ce46-9c49-47e8-900a-f92387401577
</details>

<details>
<summary> Privacy setting (double check there are no excessive items) </summary>



</details>

<details>
<summary> Private tab (make sure no UI regression) </summary>

</details>

**Comment by kjozwiak** [comment]
Labelling as `QA/No` & `relesae-notes/blocking` in favour of https://github.com/brave/brave-browser/issues/36087. However, who ever is working on the above can keep going till we get RC builds with `122.0.6261.43` which will require `1.63.156` or higher.

**Comment by hffvld** [comment]
Verified on `Galaxy Tab S8` using version(s):
```
Device/OS: Galaxy Tab S8 / gts8wifixx-user 13 TP1A.220624.014 release-keys
Brave build: 1.63.155 
Chromium: 122.0.6261.29 (Official Build) (64-bit) 
```
-------------------------

<details>
<summary> NTP layout (double check there is no UI regression) </summary>

Confirmed that `NTP` looks correct in both `Standard` and `Private` tab

|1|2|3|4|
|:-:|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/172d90f8-1acd-4b96-a79f-d2878d132f67) | ![2](https://github.com/brave/brave-browser/assets/128532965/58fdb26d-9d1c-47b9-81ee-0bd125d0a2ce) | ![3](https://github.com/brave/brave-browser/assets/128532965/6bd03ba7-d5ea-4477-bccb-807beb0627c0) | ![4](https://github.com/brave/brave-browser/assets/128532965/71b354f4-d599-4fcd-b4f2-24db6e6a35d8) |
| ![1](https://github.com/brave/brave-browser/assets/128532965/2402cb5a-6ea5-48a7-8b9f-a6e256c18f73) | ![2](https://github.com/brave/brave-browser/assets/128532965/138ed3c4-2eaa-4eeb-9cb9-628e90ffc20a) | ![3](https://github.com/brave/brave-browser/assets/128532965/e954b7d6-eb6b-42fd-b86a-d426e730993b) | ![4](https://github.com/brave/brave-browser/assets/128532965/702ab456-0915-4b24-b85b-e83560ce2708) |
</details>

<details>
<summary> VPN (double check you can subscribe and use it) </summary>

Confirmed that `VPN` is working, able to purchase a new subscription, restore existing one, and cancel a subscription

|1|2|3|4|
|:-:|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/53870966-723f-4db6-b4ae-ec3d945d0159) | ![2](https://github.com/brave/brave-browser/assets/128532965/98c3056e-4f6d-400b-9e29-4d9a165f35d0) | ![3](https://github.com/brave/brave-browser/assets/128532965/f7e65913-b4c5-454d-b394-fc54bac5be13) | ![4](https://github.com/brave/brave-browser/assets/128532965/79542ee6-ebde-4c2d-9c8b-9c108e3ad4a4) |
| ![1](https://github.com/brave/brave-browser/assets/128532965/d9576218-98ee-4561-96b1-c98844ad5781) | ![2](https://github.com/brave/brave-browser/assets/128532965/60dc2245-1775-4b81-97fc-f50ed70ba186) | ![3](https://github.com/brave/brave-browser/assets/128532965/d3821e09-4a04-4293-ba4e-69bd76bd6ebf) | ![4](https://github.com/brave/brave-browser/assets/128532965/c039839b-28b2-436d-a889-db01416b5e86) |
| ![1](https://github.com/brave/brave-browser/assets/128532965/8ade9ab9-b097-417a-bb9f-d763671b89d4) | ![2](https://github.com/brave/brave-browser/assets/128532965/a2f9124e-5293-49db-8376-cd172b81682a) | ![3](https://github.com/brave/brave-browser/assets/128532965/040d1ea0-607a-488b-9aa3-cfe1e34f4672) | ![4](https://github.com/brave/brave-browser/assets/128532965/924f1404-3eb5-4507-a251-9096aa990ed7) |
</details>

<details>
<summary> Wallet (double check there is no regression) </summary>

Confirmed `Wallet` creation and restoration are working

|1|2|3|4|
|:-:|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/ac63aa0f-8aee-47c9-8891-df3086d3b2f6) | ![2](https://github.com/brave/brave-browser/assets/128532965/18e5b9aa-3b97-4bff-b641-8a4d22221ba1) | ![3](https://github.com/brave/brave-browser/assets/128532965/438a734c-2fd8-4c4c-87f9-f0285f0b5575) | ![4](https://github.com/brave/brave-browser/assets/128532965/2cd1d1c7-7e44-450a-979c-0a9892f9c8c1) |
</details>

<details>
<summary> Playlist (double check there is no regression (if applicable)) </summary>

The playlist is not available in Brave `1.63.155` with `Chromium: 122.0.6261.29 (Official Build) (64-bit) `

|1|2|
|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/e34d20ee-498d-43f8-b878-3528865c82ed) | ![2](https://github.com/brave/brave-browser/assets/128532965/3bb1e98c-3d80-49fb-ba66-0982349ea3da) |
</details>

<details>
<summary> Widevine support (double check it works as expected) </summary>

Confirmed that media files on websites like Spotify or HBO Max can be played when `Wedivine` is ON

|1|2|3|
|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/b6241126-3585-4005-8463-43db6d032ab6) | ![2](https://github.com/brave/brave-browser/assets/128532965/96120f04-69eb-43b6-85e0-ac2ea590b18b) | ![3](https://github.com/brave/brave-browser/assets/128532965/f975ac30-6205-4efb-ba2a-e36a13efbbaf) |
| ![1](https://github.com/brave/brave-browser/assets/128532965/926cc8c9-a949-4f46-a08c-793ab5d9220d) | ![2](https://github.com/brave/brave-browser/assets/128532965/a3e82114-2742-4336-b4d2-60c6cd34c84b) | ![3](https://github.com/brave/brave-browser/assets/128532965/54dd4499-e697-4d75-89a1-1807672d178b) |
</details>

<details>
<summary> Brave Wallet (generally check it works as expected) </summary>

Confirmed that `Brave Rewards` can be enabled and `Uphold` wallet can be connected

|1|2|3|4|
|:-:|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/83610447-8c15-44e3-b12f-57cfb517fe43) | ![2](https://github.com/brave/brave-browser/assets/128532965/808c1316-bbd0-48f2-bbf2-440ea09c53f3) | ![3](https://github.com/brave/brave-browser/assets/128532965/dd05a228-3574-471e-9ad4-545e7f500ce6) | ![4](https://github.com/brave/brave-browser/assets/128532965/982abfb8-ef72-418b-b861-1b0eae50317b) |
| ![1](https://github.com/brave/brave-browser/assets/128532965/0789d02e-a48f-4b89-8720-18ab34b707b5) | ![2](https://github.com/brave/brave-browser/assets/128532965/c3acac7e-f6c3-462b-8ad4-836d02d011a2) | ![3](https://github.com/brave/brave-browser/assets/128532965/ef438e99-8d73-4196-ace3-8baa058a7933) | ![4](https://github.com/brave/brave-browser/assets/128532965/766399b3-deec-406a-928e-f501aadd58f0) |
</details>

<details>
<summary> Autoplay setting (double check it works as expected) </summary>

Confirmed that `Background` play is also working as expected when the toggle switch is ON or OFF

|1|2|
|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/5f4006d4-80f2-4b6e-bf09-bda6297ae96b) | ![2](https://github.com/brave/brave-browser/assets/128532965/88e900de-f1a3-4f72-b8a7-db60fddd0b14) |

https://github.com/brave/brave-browser/assets/128532965/760a5bc2-e2a9-41fe-8891-9a3b85cb03a4
</details>

<details>
<summary> Privacy setting (double check there are no excessive items) </summary>

Confirmed that `Brave Shields & privacy` settings are correct, changing the settings is working and not causing Brave to crash

|1|2|3|
|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/fd3539b8-0cfb-4295-8a88-64ce224e983d) | ![2](https://github.com/brave/brave-browser/assets/128532965/56392ec8-98cd-457c-8739-d1a55d569cd9) | ![3](https://github.com/brave/brave-browser/assets/128532965/a82554b6-23e7-4421-a62a-e2d4849b7432) |
| ![1](https://github.com/brave/brave-browser/assets/128532965/4b8d786b-1a78-43f2-b64f-77dc9c552368) | ![2](https://github.com/brave/brave-browser/assets/128532965/287e1b9c-7998-4041-84cb-8771b25a6671) | ![3](https://github.com/brave/brave-browser/assets/128532965/52aa1269-2187-47c8-9982-d1282251b3d7) |
</details>

<details>
<summary> Private tab (make sure no UI regression) </summary>

Confirmed that the `Private` tab is working, the `Private` tab `Lock/Unlock` is working, Non-default SE is working

|1|2|3|
|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/eb9474d0-4980-4136-bb45-82a1592b3250) | ![2](https://github.com/brave/brave-browser/assets/128532965/1eddaaf1-efb2-4c97-9be5-ba65ef2e2a28) | ![3](https://github.com/brave/brave-browser/assets/128532965/9d3680eb-c6f8-4fd3-aa5e-2de49c45bb94) |
| ![1](https://github.com/brave/brave-browser/assets/128532965/ea0902ae-817a-48a4-bf02-38b80d1d4ee6) | ![2](https://github.com/brave/brave-browser/assets/128532965/8aedf1fb-58b7-4d83-8526-36757d810e91) | ![3](https://github.com/brave/brave-browser/assets/128532965/bd0ba608-b827-4cac-8774-ceab824c56fb) |
</details>
